requirements-max.txt

```diff
@@ -1,4 +1,3 @@
 # this requirements file applies upper pins on critical upstream dependencies with incompatible functionality / refactoring that has not yet been implemented in this repository
 
 stcal<1.4.0
-stdatamodels<1.4.1
```

Comment:
`stdatamodels 1.5.0` is now released, so this could be updated to `stdatamodels<1.6.0`

Reply:
✅